index.html

```diff
@@ -2138,8 +2138,14 @@ <h3 id="Schema-Building">Schema Building:</h3>
       You can also chain <tt>onDelete</tt> and/or <tt>onUpdate</tt> to set the reference option (<tt>RESTRICT</tt>, <tt>CASCADE</tt>,
       <tt>SET NULL</tt>, <tt>NO ACTION</tt>) for the operation.  Note, this is the same as <code>column.references(column)</code>
       but works for existing columns.
+    <pre class="display">
+    knex.table('users');
```

Comment:
`knex.table('users')` is a bit misleading, since it should actually be

```
knex.schema.table('users', function (table) {
  ...
})
```

Looks like the other examples are just directly using `table.someBuilderFunction(...)` so would be enough / less misleading to just remove the line `knex.table('users');` from the example.